attributes/default.rb

```diff
@@ -23,3 +23,42 @@
 default['nexus_repository_manager']['nexus']['data']['etc']['path'] = default['nexus_repository_manager']['nexus']['data']['path'] + '/etc'
 default['nexus_repository_manager']['nexus']['data']['log']['path'] = default['nexus_repository_manager']['nexus']['data']['path'] + '/log'
 default['nexus_repository_manager']['nexus']['data']['tmp']['path'] = default['nexus_repository_manager']['nexus']['data']['path'] + '/tmp'
+
+# hazelcast configuration
+## group configuration
+default['nexus_repository_manager']['hazelcast']['group']['name'] = 'nexus'
+default['nexus_repository_manager']['hazelcast']['group']['password'] = 'nexus123'
+
+## multicast configuration
+default['nexus_repository_manager']['hazelcast']['network']['join']['multicast']['enabled'] = true
+default['nexus_repository_manager']['hazelcast']['network']['join']['multicast']['multicast_group'] = '224.2.2.3'
+default['nexus_repository_manager']['hazelcast']['network']['join']['multicast']['multicast_port'] = 54327
+
+## tcp-ip configuration
+default['nexus_repository_manager']['hazelcast']['network']['join']['tcp_ip']['enabled'] = false
+default['nexus_repository_manager']['hazelcast']['network']['join']['tcp_ip']['interface'] = '127.0.0.1'
+default['nexus_repository_manager']['hazelcast']['network']['join']['tcp_ip']['members'] = ['127.0.0.1']
+
+## aws configuration
+default['nexus_repository_manager']['hazelcast']['network']['join']['aws']['enabled'] = false
+default['nexus_repository_manager']['hazelcast']['network']['join']['aws']['iam_role'] = ''
+default['nexus_repository_manager']['hazelcast']['network']['join']['aws']['region'] = 'us-east-1'
+default['nexus_repository_manager']['hazelcast']['network']['join']['aws']['tag_key'] = 'Platform'
+default['nexus_repository_manager']['hazelcast']['network']['join']['aws']['tag_value'] = 'nexus-repository-manager'
+
+## hazelcast discovery configuration
+default['nexus_repository_manager']['properties']['hazelcast']['discovery']['enabled'] = false
```

Comment:
it looks like properties and hazelcast is swapped around here

Reply:
^

Reply:
Flipped hazelcast and properties. This was done on purpose to match the xml nodes to "reduce" confusion. Guess that did not work :) (cd8305d)